extensions/amp-consent/0.1/amp-consent.css

```diff
@@ -41,19 +41,35 @@ amp-consent.amp-hidden {
   visibility: hidden;
 }
 
-amp-consent.i-amphtml-consent-loading {
-  /* TODO: Finalize the placeholder design */
+@keyframes loading-placeholder-spin  {
```

Comment:
maybe prefix with amp-consent into the name of this, since keyframe names are global